### PR TITLE
SAK-32160 Use Font-Awesome icons in content review implementations

### PR DESCRIPTION
**Pull request by josecebe**
Use Font-Awesome icons in content review implementations

### CODE REVIEW

**Comment by ottenhoff** [comment]
I don't think alt is a valid property on <span> tags

**Comment by josecebe** [comment]
I forgot to remove it when I changed it from  to <span>, pull request updated.

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm [hunk 1]
```diff
@@ -709,10 +709,14 @@
 											#set ($props = $reviewResult.getContentResource().Properties)
 											<span class="reportIcon">
 											#if ($reviewResult.getReviewReport().equals("Error"))
-												<img src="/library/skin/images/error.png" alt="$reviewResult.getReviewError()" title="$reviewResult.getReviewError()">
+												<span class="fa fa-exclamation-triangle contentReviewWarn" alt="$reviewResult.getReviewError()" title="$reviewResult.getReviewError()"></span>
```
Comment (ottenhoff) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm]: alt tag on span

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm [hunk 2]
```diff
@@ -709,10 +709,14 @@
...
 											#else
 												<a href="$reviewResult.getReviewReport()" target="_blank">
+												#if (!$reviewResult.getReviewIconCssClass().isEmpty() && !$reviewResult.getReviewIconStyle().isEmpty())
+													<span class="$reviewResult.getReviewIconCssClass()" style="$reviewResult.getReviewIconStyle()" alt="$reviewResult.getReviewScore()" title="$reviewResult.getReviewScore()"></span>
```
Comment (ottenhoff) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm]: style and alt tag on span

**Comment by mpellicer** [comment]
PR updated. Also contains a good improvement, the threshold colors & icons definition were moved to Assignment's CSS file, easier to customize.

**Comment by jonespm** [comment]
Yeah cool idea. I'd probably NPE check on score on the getIconCssClassforScore methods.

Though I don't see the point of even using a Long score, using a long score or an int score makes more sense. Since what the code is doing now is converting an int to a Long to accommodate this method.

**Comment by ern** [comment]
If the old Icon api is no longer needed can you remove... thank you!

File: content-review/impl/compilatio/src/main/java/org/sakaiproject/contentreview/compilatio/CompilatioReviewServiceImpl.java [hunk 3]
```diff
@@ -716,6 +716,21 @@ public String getIconUrlforScore(Long score) {
 	}
 
 	@Override
+	public String getIconCssClassforScore(Long score) {
+		if (score.equals(Long.valueOf(0))) {
```
Comment (buckett) [inline content-review/impl/compilatio/src/main/java/org/sakaiproject/contentreview/compilatio/CompilatioReviewServiceImpl.java]: Could we switch the code a little (as @jonespm suggested switch to primatives) and make these:

```
if (score == 0) {
  return ...
} else if ( score < 25) {
  return ...
} else if (score < 50) {
  return ...
} else if (score < 75) {
```
As it's much easier to read the code as long as the values only ever increase.

**Comment by josecebe** [comment]
PR updated with all the recommended changes

**Pull request merged**